libbeat/idxmgmt/ilm/ilm_test.go

```diff
@@ -287,8 +287,9 @@ func TestDefaultSupport_Manager_EnsurePolicy(t *testing.T) {
 				onHasILMPolicy(testPolicy.Name).Return(true, nil),
 			},
 		},
-		"overwrite existing": {
+		"overwrite": {
```

Comment:
The test was wrong. When overwrite is enabled we ignore checkExists and always attempt to create the resource. The mock as configured checks that: HasPolicy will not be called, ILMPolicy will be created and indicates success.